packages/libssp.rb

```diff
@@ -10,10 +10,10 @@ class Libssp < Package
   source_sha256 '64f404c1a650f27fc33da242e1f2df54952e3963a49e06e73f6940f3223ac344'
 
   binary_url({
-    aarch64: 'https://github.com/chromebrew/binaries/raw/main/armv7l/libssp-10.3.0-chromeos-armv7l.tar.xz',
-     armv7l: 'https://github.com/chromebrew/binaries/raw/main/armv7l/libssp-10.3.0-chromeos-armv7l.tar.xz',
-       i686: 'https://github.com/chromebrew/binaries/raw/main/i686/libssp-10.3.0-chromeos-i686.tar.xz',
-     x86_64: 'https://github.com/chromebrew/binaries/raw/main/x86_64/libssp-10.3.0-chromeos-x86_64.tar.xz'
+    aarch64: 'https://github.com/chromebrew/binaries/releases/download/v_gcc10_10.3.0/libssp-10.3.0-chromeos-armv7l.tar.xz',
+     armv7l: 'https://github.com/chromebrew/binaries/releases/download/v_gcc10_10.3.0/libssp-10.3.0-chromeos-armv7l.tar.xz',
+       i686: 'https://github.com/chromebrew/binaries/releases/download/v_gcc10_10.3.0/libssp-10.3.0-chromeos-i686.tar.xz',
+     x86_64: 'https://github.com/chromebrew/binaries/releases/download/v_gcc10_10.3.0/libssp-10.3.0-chromeos-x86_64.tar.xz'
```

Comment:
I'm not sold on the releases idea.  This is going to make the repo huge over time.  As you know, space costs.

Reply:
Discussion on slack...